packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -147,15 +240,58 @@ export function useApolloUsageReport(options: ApolloUsageReportOptions = {}): Pl
               return;
             }
 
-            const operationName =
-              context.params.operationName ??
-              (isDocumentNode(result) ? getOperationAST(result)?.name?.value : undefined);
-            const signature = context.params.query
-              ? stripIgnoredCharacters(context.params.query)
-              : '';
+            ctx.schemaId = currentSchema!.id;
+
+            if (isDocumentNode(result)) {
+              if (getOperationAST(result, context.params.operationName)) {
+                return;
+              } 
+                ctx.operationKey = `## GraphQLUnknownOperationName\n`;
```

Comment:
In my opinion, this will never happen and in all cases it will be overridden in the onValidate hook.

Reply:
It happens when you provide a `operationName` in the body of your request that doesn't actually exists.

When this happens, the result of parsing phase is a `DocumentNode`, but `onValidate` never gets called. So I had to check for it here.

There is a unit test to verify that this case is covered properly :-)